src/pages/CompanyPage.jsx

```diff
@@ -1,9 +1,13 @@
-function CompanyPage() {
-  return (
+import React from "react";
+
+
+function CompanyPage(props) {
+  const {companies} = props;
+  const {companySlug} = useParams();
+
+  return(
     <div>
-      <h1>CompanyPage</h1>
+      
     </div>
-  );
-}
-
-export default CompanyPage;
+  )
+}
```

Comment:
You were missing
`export default CompanyPage;`

You could find that in the terminal error when doing `npm run dev`, or in the console when opening your app